apps/admin-gui/src/app/admin/pages/admin-page/admin-attributes/admin-attributes.component.ts

```diff
@@ -75,7 +75,7 @@ export class AdminAttributesComponent implements OnInit {
   refreshTable() {
     this.loading = true;
     this.attributesManager.getAllAttributeDefinitions().subscribe(attrDefs => {
-      this.attrDefinitions = filterCoreAttributesDefinitions(attrDefs);
```

Comment:
then the function "filterCoreAttributesDefinitions" can be deleted from imports and since it is not used anywhere it can be completely deleted from perun-utils right?

Reply:
Yes, good point. I have removed it.